vnext/ReactUWP/Views/DynamicAutomationPeer.cpp

```diff
@@ -129,6 +130,26 @@ winrt::IInspectable DynamicAutomationPeer::GetPatternCore(winrt::PatternInterfac
       (HasAccessibilityState(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityStates::Expanded) ||
        HasAccessibilityState(winrt::PROJECT_ROOT_NAMESPACE::AccessibilityStates::Collapsed))) {
     return *this;
+  } else if (
+      patternInterface == winrt::PatternInterface::Value &&
+
+      // TODO
+      // Should I be limiting this at al? If so, what elements to include apart from this one?
+      accessibilityRole == winrt::PROJECT_ROOT_NAMESPACE::AccessibilityRoles::ComboBox &&
```

Comment:
From the msdn docs:

Exposes methods and properties to support access by a Microsoft UI Automation client to controls that have an intrinsic value that does not span a range and that can be represented as a string

see:  [https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.provider.ivalueprovider?view=winrt-19041](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.automation.provider.ivalueprovider?view=winrt-19041)

From that definition I would say if we have a text accessibility value and one of the roles other than the 3 identified below for range values, then it meets the criteria.

Reply:
Good point, I was thinking along similar lines, but wanted to double-check :)

Reply:
If someone set the value, then it meets the criteria and return it.